eng/Build.props

```diff
@@ -4,14 +4,16 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(BuildAllProjects)' == 'true' ">
-    <BuildNative>true</BuildNative>
-    <BuildManaged>true</BuildManaged>
-    <BuildNodeJS>true</BuildNodeJS>
-    <BuildJava>true</BuildJava>
+    <BuildNative Condition="'$(BuildNative)' == ''">true</BuildNative>
+    <BuildManaged Condition="'$(BuildManaged)' == ''">true</BuildManaged>
+    <BuildNodeJS Condition="'$(BuildNodeJS)' == ''">true</BuildNodeJS>
+    <BuildJava Condition="'$(BuildJava)' == ''">true</BuildJava>
   </PropertyGroup>
 
   <!-- These projects are always excluded, even when -projects is specified on command line. -->
   <ItemGroup>
+    <!-- Explicitly excluded projects -->
+    <ProjectToExclude Include="$(ProjectToExclude)" />
```

Comment:
The `$(ProjectToExclude)` property doesn't appear to be used. What's it for?

Reply:
I found it particularly useful to be able to explicitly exclude a subset of projects when running normal builds. You would use this like so: ./build.cmd /p:ProjectToExclude="{ get pwd }/src/Mvc/**/*.csproj;{ get pwd }/src/Middleware/**/*.csproj". I could remove it if you really hate it but otherwise I find it to be quite useful. After all, we have the ability to explicitly specify projects to build, so we should also have the ability to explicitly exclude projects.